docker/entrypoint.sh

```
@@ -1,9 +1,11 @@
 #!/bin/bash
-set -e
+set -Eeuo pipefail
 
 # Gunicorn Prometheus Exporter Sidecar Entrypoint
 # This script handles the container startup and provides different modes
 
+echo "DEBUG: entrypoint.sh started with arguments: $@"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)